rest/src/main/scala/io/udash/rest/raw/RestRequest.scala

```diff
@@ -56,6 +56,7 @@ case class HttpErrorException(code: Int, payload: HttpBody = HttpBody.Empty, cau
 object HttpErrorException {
   def plain(code: Int, message: String, cause: Throwable = null): HttpErrorException =
     HttpErrorException(code, HttpBody.plain(message), cause)
+  val Streaming: HttpErrorException = plain(400, "HTTP stream failure")
```

Comment:
Used only in client implementation, consider moving this constant to `JettyRestClient`